quickstarts/sonarqube/config.yml

```diff
@@ -0,0 +1,44 @@
+
+slug: sonarqube
+description: |
+  ## Comprehensive monitoring quickstart for SonarQube
+  
+  By integrating SonarQube with New Relic, you can get comprehensive information about the quality of your codebase and receive alerts about serious vulnerabilities. You can keep track of your code deployments as well as the complete CI/CD workflow by visualizing and gathering metrics and logs
```

Comment:
```suggestion
  By integrating SonarQube with New Relic, you can get comprehensive information about the quality of your codebase and receive alerts about serious vulnerabilities. You can keep track of your code deployments as well as the complete CI/CD workflow by visualizing and gathering metrics and logs.
```

Reply:
Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...